src/components/forms/forms.jsx

```diff
@@ -0,0 +1,150 @@
+import React, { useState } from 'react';
+import { connect } from 'react-redux';
+
+import {
+  getFormsDataThunk,
+  deleteFormSchemaThunk,
+  saveFormDataThunk
+} from './../../redux/forms-Reducer';
+import style from './forms.module.css';
+
+function Forms(props) {
+  const [state, setState] = useState({});
+
+  if (!props.checked) {
+    props.getFormsDataThunk();
+  } else if (!state.forms) setState({ forms: [...props.forms] });
```

Comment:
don't write together -> multiline ifs and inline ifs

```suggestion
  } else if (!state.forms) {
    setState({ forms: [...props.forms] });
  }
```